scenes/cslib_object.txt

```diff
@@ -0,0 +1,111 @@
+*comment cslib_object
+*comment Allows the simple manipulation of arrays of sets of variables
+*comment @StephenHart
+*comment
+*comment How it works:
+*comment A object is always accessed through its main instance.
+*comment When you "get" an instance of a object, the data for that particular instance 
+*comment is copied into the main instance
+*comment
+*comment A object definition has the following elements defined in startup.txt:
+*comment	1. A object "fieldname" array defining all the field names
+*comment		objectname_fieldname_N fieldname
+*comment	2. A main instance of this object. This is used to 'return'
+*comment		the object (similar to cslib_ret). Also includes the index
```

Comment:
"Also includes the index"? Does it? Your example doesn't show this.
That said, in this simple version of the module, I don't think there is a need to return the ID, because you will always have already had it when making the call (whether set or get). So you probably just remove this comment.

Reply:
Comment accidentally got left in from an earlier draft. I'll remove it.